package.json

```diff
@@ -33,10 +33,12 @@
     "@material-ui/icons": "^4.5.1",
     "@material-ui/styles": "^4.5.0",
     "axios": "^0.19.0",
+    "node-sass": "^4.12.0",
```

Comment:
We are no longer using SASS for styling. We are using [CSS-in-JSS](https://material-ui.com/styles/basics/#why-use-material-uis-styling-solution) instead. `node-sass` is no longer needed.

Reply:
ok, I will remove this from my dependencies